slaves/linux-cross/README.md

```diff
@@ -165,6 +165,7 @@ For targets: `mipsel-unknown-linux-musl`
 - Target options > Target Architecture = mips
 - Target options > Endianness = **Little** endian
 - Target options > Architecture level = mips32 -- (+)
+- Target options > Floating point = software (no FPU) -- (+)
```

Comment:
Not sure whether or not that `(+)` is correct here, this line was basically copied from the `mips-unknown-linux-musl` target.